tests/resources/xnys.csv

```diff
@@ -8313,4 +8313,254 @@
 2022-12-27,2022-12-27 14:31:00,2022-12-27 21:00:00
 2022-12-28,2022-12-28 14:31:00,2022-12-28 21:00:00
 2022-12-29,2022-12-29 14:31:00,2022-12-29 21:00:00
-2022-12-30,2022-12-30 14:31:00,2022-12-30 21:00:00
\ No newline at end of file
+2022-12-30,2022-12-30 14:31:00,2022-12-30 21:00:00
```

Comment:
@sercant Just for interest how did you collect and write these dates 😅

Reply:
There is util code for it :)

https://github.com/AlpacaTechJP/trading_calendars/blob/master/etc/make_trading_calendar_test_csv.py

Reply:
Oh I think I might have used this script when I was doing the same for exchange_calendars 😅 it has been a while so I forgot